### DISCUSSION TITLE
Can I use a "slug" in my stock location URLs, instead of ID? E.g. /stock/location/ethernet-cables-box instead of /stock/location/3/

### DISCUSSION THREAD

**Discussion by ndbroadbent** [inventree/InvenTree#5316] [Q&A]
I'm generating some QR codes for printable labels that I'm going to stick on some storage boxes. I thought it would be cool if I could scan the QR code to open the stock location page in InvenTree and see all the parts in the box, update parts, etc.

It would be nice if I could refer to a "slug" in the URL, like `/stock/location/ethernet-cables-box` instead of `/stock/location/3`. I feel like this would be more flexible and might prevent me from needing to reprint the labels in the future.

Is this possible?

**Comment by SchrodingersGat** [comment]
@ndbroadbent you cannot access items via a slug in this fashion - also the ID is immutable (so "3" will always point to the same location)

You can include the name of the location (or whatever informat you want) into the labels, but the best idea is to include the ID in the barcode, or use the built-in barcode types to allow 1:1 association
